tests/artificial_viscosity_list/solution/solution-00001.0000.gnuplot

```diff
@@ -5,21 +5,21 @@
 # For a description of the GNUPLOT format see the GNUPLOT manual.
 #
 # <x> <y> <velocity> <velocity> <p> <T> <C_1> <artificial_viscosity> <artificial_viscosity_composition> 
-6.336e+06 0 0 1.36111e-11 -2.67379 973 1 0 0.000152425 
-6.12011e+06 1.63988e+06 -1.34172e-09 5.00735e-09 2.75252 973 0.999998 0 0.000152425 
+6.336e+06 0 0 1.35985e-11 -2.67379 973 1 0 0.000152425 
```

Comment:
do you know why this changes?

Reply:
It doesn't change by much, so my suspicion is that it may just be due to the change in the order we compute things in. I think all of the tests that change are with the simple model and have at least one compositional field (but not necessarily one that changes the density). Before, we computed the density as rho = rho(T) + Delta_rho * C. Now we compute the density as rho0 = rho(T), rho1=rho(T) + Delta_rho, rho=average(rho0, rho1). 

Analytically, the old and the new computations should be the same, but numerically, there could be small differences that lead to small changes in iteration count etc. as we see in the tests.

Reply:
I looked into this a bit more, and there are cookbooks that use the simple model, compositional fields, and a density difference for that compositional field, that remain unchanged (like point_value_01, which is a variation of the sinker benchmark). So I think the density computation is correct.

Reply:
Hm, ok.